error[E0277]: the trait bound `NetDefn: Default` is not satisfied
   --> crates/tor-geoip/src/lib.rs:271:21
    |
271 |             map_v4: Default::default(),
    |                     ^^^^^^^^^^^^^^^^ the trait `Default` is not implemented for `NetDefn`
    |
    = help: the trait `Default` is implemented for `RangeInclusiveMap<K, V, StepFnsT>`
    = note: required for `RangeInclusiveMap<u32, NetDefn>` to implement `Default`
help: consider annotating `NetDefn` with `#[derive(Default)]`
    |
214 + #[derive(Default)]
215 | struct NetDefn {
    |

error[E0277]: the trait bound `NetDefn: Default` is not satisfied
   --> crates/tor-geoip/src/lib.rs:272:21
    |
272 |             map_v6: Default::default(),
    |                     ^^^^^^^^^^^^^^^^ the trait `Default` is not implemented for `NetDefn`
    |
    = help: the trait `Default` is implemented for `RangeInclusiveMap<K, V, StepFnsT>`
    = note: required for `RangeInclusiveMap<u128, NetDefn>` to implement `Default`
help: consider annotating `NetDefn` with `#[derive(Default)]`
    |
214 + #[derive(Default)]
215 | struct NetDefn {
    |

For more information about this error, try `rustc --explain E0277`.
error: could not compile `tor-geoip` (lib) due to 2 previous errors

--- a/crates/tor-geoip/src/lib.rs
+++ b/crates/tor-geoip/src/lib.rs
@@ -210,7 +210,7 @@ impl Display for OptionCc {
 /// A country code / ASN definition.
 ///
 /// Type lifted from `geoip-db-tool` in the C-tor source.
-#[derive(Copy, Clone, Eq, PartialEq, Debug)]
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Default)]
 struct NetDefn {
     /// The country code.
     ///
